aioredis has changed and no longer works in CI

```
[2947] /home/runner/work/python-dockerflow/python-dockerflow$ /home/runner/work/python-dockerflow/python-dockerflow/.tox/py37-s20/bin/python -m pip freeze >.tox/py37-s20/log/py37-s20-3.log
py37-s20 installed: aiofiles==0.6.0,aiohttp==4.0.0a1,aioredis==2.0.0a1,async-timeout==3.0.1,attrs==20.3.0,certifi==2020.12.5,chardet==3.0.4,coverage==5.6b1,-e git+https://github.com/mozilla-services/python-dockerflow@0878ba3264b10636c068eb6b5d189780d6ff3adf#egg=dockerflow,fakeredis==1.5.0,h11==0.9.0,httpcore==0.11.1,httptools==0.1.1,httpx==0.15.4,idna==3.1,importlib-metadata==4.0.1,iniconfig==1.1.1,jsonschema==3.2.0,mock==4.0.3,multidict==4.7.6,packaging==20.9,pluggy==1.0.0.dev0,py==1.10.0,pyparsing==3.0.0b2,pyrsistent==0.17.3,pytest==6.2.3,pytest-cov==2.11.1,pytest-cover==3.0.0,pytest-coverage==0.0,pytest-mock==3.5.1,pytest-pythonpath==0.7.3,redis==3.1.0,rfc3986==1.4.0,sanic==20.9.0,sanic-redis==0.2.0,six==1.15.0,sniffio==1.2.0,sortedcontainers==2.3.0,toml==0.10.2,typing-extensions==3.7.4.3,ujson==4.0.2,uvloop==0.15.2,websockets==8.1,yarl==1.6.3,zipp==3.4.1
  E   ImportError: cannot import name 'create_redis_pool' from 'aioredis' (/home/runner/work/python-dockerflow/python-dockerflow/.tox/py37-s20/lib/python3.7/site-packages/aioredis/__init__.py)
```

--- a/tests/requirements/sanic.txt
+++ b/tests/requirements/sanic.txt
@@ -1,7 +1,7 @@
 # these are constrained by the files in tests/constraints/*.txt
 # to support a triple stack Django/Flask/Sanic
 aiohttp
-aioredis
+aioredis<2.0.0
 Sanic
 sanic_redis
 uvloop>=0.14.0rc1

